doc/sphinx/source/recipes/recipe_pv_capacity_factor.rst

```diff
@@ -0,0 +1,73 @@
+.. _recipes_pv_capacity_factor:
+
+Capacity factor for solar photovoltaic (PV) systems
+===================================================
+
+Overview
+--------
+
+This diagnostic computes the PV factor, a measure of the fraction of the 
+maximum possible energy produced per PV grid cell. It uses the daily incoming 
+surface solar radiation and the surface temperature with a method described
+in `Bett and Thornton (2016)`_. The user can select temporal
+range, season, and region of interest.
+
+
+.. _`Bett and Thornton (2016)`: https://doi.org/10.1016/j.renene.2015.10.006
+
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in recipes/
+
+* recipe_pv_capacity_factor.yml
+
+Diagnostics are stored in diag_scripts/pv_capacityfactor/
+
+* pv_capacity_factor.R: prepare data and plots results.
+* PV_CF.R: calculates the daily capacity factor.
+
+
+User settings
+-------------
+
+User setting files are stored in recipes/
+
+#. recipe_capacity_factor.yml
+
+   *Required settings for script*
+
+   * season: String to include shortcut for Season in plot title and name (e.g. "djf"). It will be converted to upper case. 
+   
```

Comment:
Ok, the season here is actually not selecting anything, just used as a string in the figure title and file name. I wrote some more to clarify this:
* season: String to include shortcut for season in plot title and name (e.g. "djf").
     It will be converted to upper case. This season should be the one set in the preprocessor,
     since it is only used as a string and does not affect the data in the diagnostic.
     In the default recipe this is solved through a node anchor.